docs_src/examples/Ch-ExamplesAddingMQTTDevice.md

```diff
@@ -197,6 +197,10 @@ topics, make the following changes in the device service configuration files:
        # Uncomment below to use multi-level topics
        CommandTopic = "command/my-custom-device"
    ```
+   !!! note
+          If you have run Device-MQTT before, you will need to delete the previously registered device(s) using a command similar to the one below:
+          `curl --request DELETE http://localhost:59881/api/v2/device/name/MQTT-test-device`
```

Comment:
```suggestion
          `curl --request DELETE http://localhost:59881/api/v2/device/name/my-custom-device`
```

Reply:
I thought that the delete would be used for removing the default device (if the user already built/ran the project). Changing it would remove a previously loaded custom device, which is what we are saying to add.

Reply:
Ahh, yes. I see now. Ignore this. :-)